.github/workflows/wheels.yml

```diff
@@ -57,63 +57,66 @@ jobs:
           # https://github.com/scikit-learn/scikit-learn/issues/22530
           - os: windows-2019
             python: 38
-            bitness: 64
             platform_id: win_amd64
           - os: windows-latest
             python: 39
-            bitness: 64
             platform_id: win_amd64
           - os: windows-latest
             python: 310
-            bitness: 64
+            platform_id: win_amd64
+          - os: windows-latest
+            python: 311
             platform_id: win_amd64
 
           # Linux 64 bit manylinux2014
           - os: ubuntu-latest
             python: 38
-            bitness: 64
             platform_id: manylinux_x86_64
             manylinux_image: manylinux2014
           - os: ubuntu-latest
             python: 39
-            bitness: 64
             platform_id: manylinux_x86_64
             manylinux_image: manylinux2014
 
           # NumPy on Python 3.10 only supports 64bit and is only available with manylinux2014
           - os: ubuntu-latest
             python: 310
-            bitness: 64
+            platform_id: manylinux_x86_64
+            manylinux_image: manylinux2014
+
+          # NumPy on Python 3.11 only supports 64bit and is only available with manylinux2014
```

Comment:
I guess we can now remove both comment if we only distribution wheels for 64bit architectures.